dev/io.openliberty.xmlBinding.3.0.internal.tools/src/io/openliberty/xmlbinding/tools/internal/JaxbToolsConstants.java

```diff
@@ -21,7 +21,7 @@
 
     public static final String TR_GROUP = "JaxbTools";
 
-    public static final String TR_RESOURCE_BUNDLE = "io.openliberty.jaxb.tools.internal.resources.JaxbToolsMessages";
+    public static final String TR_RESOURCE_BUNDLE = "io.openliberty.xmlbinding.tools.internal.resources.JaxbToolsMessages";
```

Comment:
You changed this here, but the file is still io.openliberty.jaxb in reality.  Also JaxbToolsMessages should be changed as well it would seem.

Reply:
Should the TR_GROUP also be updated?

Reply:
I think this is going a bit too far. There are still plenty of classes that start with `Jaxb` in the XML Binding APIs.  If the spec project didn't need to rename their classes, I don't see why we would have to.